webhook_server/tests/test_issue_comment_handler.py

```
@@ -136,7 +144,12 @@ async def test_process_comment_webhook_data_parallel_execution(
         # Track execution order and timing
         execution_events: list[tuple[str, str, float]] = []  # (command, event, timestamp)
 
-        async def mock_command(pull_request, command, reviewed_user, issue_comment_id):
+        async def mock_command(  # noqa: ARG001
```

Ruff (0.14.11)

Unused `noqa` directive (unused: `ARG001`)

Remove unused `noqa` directive

(RUF100)